README.md

```diff
@@ -32,23 +32,37 @@ For more information, see the [official lakeFS documentation](https://docs.lakef
 
 ## Capabilities
 
-### In Development
+### ETL Testing with Isolated Dev/Test Environment
 
-* **Experiment** - try new tools, upgrade versions, and evaluate code changes in isolation. You can get an isolated snapshot to run and compare experiments by creating a new branch of the data, while others are not exposed.
-* **Debug** - checkout specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
-* **Collaborate** - leverage isolated branches managed by metadata (not copies of files) to work in parallel.
+When working with a data lake, it’s useful to have replicas of your production environment. These replicas allow you to test these ETLs and understand changes to your data without impacting the consumers of the production data.
```

Comment:
replace this - " without impacting the consumers of the production data."
with:
without impacting downstream data consumers.

Reply:
Thank you. Added the change.